tests/tests_mqtt_server/src/server.rs

```diff
@@ -0,0 +1,66 @@
+use once_cell::sync::OnceCell;
+use std::{
+    net::TcpStream,
+    process::{Child, Command, Stdio},
+    sync::{
+        atomic::{AtomicUsize, Ordering},
+        Mutex,
+    },
+    thread::sleep,
+    time::Duration,
+};
+
+static SERVER: OnceCell<Mutex<Child>> = OnceCell::new();
```

Comment:
Moving these static fields and static functions into the `TestsMqttServer` impl would have been cleaner, without needing all these atomic structures shared and all, right? If the intention was to have a single server running in the background while multiple tests parallelly run against it, then this might be the only way (I assume).